plugins/reports/counter/classes/releases/CounterRelease3_0.inc.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file plugins/generic/sushiLite/classes/releases/CounterRelease3_0.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class CounterRelease3_0
+ * @ingroup plugins_reports_counter
+ *
+ * @brief A COUNTER report, release 3.0
+ */
+
+import('plugins.reports.counter.classes.releases.CounterRelease');
+
+class CounterRelease3_0 extends CounterRelease {
```

Comment:
it would make much more sense to have CounterRelease3_0 extends CounterReport, in my opinion. Right now it seems that we have a CounterRelease opposed to a CounterReport object, which is not true. Everything is a CounterReport, but we use some base classes to wrap reports specific to releases, which is absolutely fine. Just a naming thing.